### PR TITLE
feat(figma-widget): create guidance json files 

### PR DESCRIPTION
**Pull request by valentin-mladenov**
CDE-2334

### CODE REVIEW

File: 1001-no-hardcoded-colors.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+updated: 2014-10-22
+---
+
+# Clarity design colors
+
+Design teams consuming Clarity Components in figma.
+
+## Guidance
+
+- Design teams **Must** use color tokens provided trough Clarity Components.
```
Comment (jbusta5) [inline 1001-no-hardcoded-colors.md:14]: Design teams **Must** use color tokens from Clarity components to keep designs consistent and on-brand.

File: 1002-no-detached-design-components.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+updated: 2014-10-22
+---
+
+# Clarity design components 
+
+Design teams consuming Clarity Components in figma.
+
+## Guidance
+
+- Design teams **Must not** detach Clarity Components in figma.
```
Comment (jbusta5) [inline 1002-no-detached-design-components.md:14]: Design teams **Must Not** detach Clarity components in Figma. Doing so breaks alignment with the design system and makes it harder to maintain consistency.

File: 1001-no-hardcoded-colors.md (hunk 1 above)
Comment (valentin-mladenov) [reply 1001-no-hardcoded-colors.md]: moved to PR #119

**Comment by valentin-mladenov** [comment]
merge before vmware-clarity/plugins#2

**Review by Jinnie — APPROVED** [review]
LGTM

**Pull request merged**